plugin/trino-hive/src/main/java/io/trino/plugin/hive/ion/IonPageSourceFactory.java

```diff
@@ -61,6 +79,18 @@ public class IonPageSourceFactory
     // this is used as a feature flag to enable Ion native trino integration
     private final boolean nativeTrinoEnabled;
 
+    private static final Map<String, String> EXACT_PROPERTIES = ImmutableMap.of(
+            FAIL_ON_OVERFLOW_PROPERTY, FAIL_ON_OVERFLOW_PROPERTY_DEFAULT,
+            IGNORE_MALFORMED, IGNORE_MALFORMED_DEFAULT,
+            PATH_EXTRACTION_CASE_SENSITIVITY, PATH_EXTRACTION_CASE_SENSITIVITY_DEFAULT,
+            ION_TIMESTAMP_OFFSET_PROPERTY, ION_TIMESTAMP_OFFSET_DEFAULT,
+            ION_SERIALIZATION_AS_NULL_PROPERTY, ION_SERIALIZATION_AS_NULL_DEFAULT);
+
+    private static final Set<Pattern> PATTERN_PROPERTIES = ImmutableSet.of(
```

Comment:
These are all "Column" specific right? I would call them `COLUMN_PROPERTIES`

Reply:
Yes, I will update the name.